server/test/api/subsidy.js

```diff
@@ -0,0 +1,112 @@
+/*global describe, it, beforeEach, process*/
+
+var chai = require('chai');
+var chaiHttp = require('chai-http');
+var expect = chai.expect;
+chai.use(chaiHttp);
+
+var helpers = require('./helpers');
+helpers.configure(chai);
+
+var url = 'https://localhost:8080';
+var user = { username : 'superuser', password : 'superuser', project: 1};
+
+describe('The subsidy API, PATH : /subsidies', function () {
+  var agent = chai.request.agent(helpers.baseUrl);
+
+  var newSubsidy = {
+    account_id : 3626,
+    label : 'tested subsidy',
+    description : 'This is for the test',
+    value : 10  
+  };
+ 
+  var wrongSubsidy = {
+    account_id : 3626,
+    label : 'tested wrong subsidy',
+    description : 'This is for the wrong value test',
+    value : -10  
+  };
+
+  beforeEach(helpers.login(agent));
+
+  it('METHOD : POST, PATH : /subsidies, It adds a subsidy', function () {
+    return agent.post('/subsidies')
+      .send(newSubsidy)
+      .then(function (res) {
+        expect(res).to.have.status(201);
+        expect(res).to.be.json;
+        expect(res.body).to.not.be.empty;
+        expect(res.body.id).to.be.defined;
+        newSubsidy.id = res.body.id;
+        return agent.get('/subsidies/' + newSubsidy.id);  
+      })
+      .then(function (res){ 
+        expect(res).to.have.status(200);
+        expect(res.body).to.have.all.keys('id', 'account_id', 'label', 'description', 'value', 'created_at', 'updated_at');
+      })
+     .catch(helpers.handler);
+  });
+
+  it('METHOD : POST, PATH : /subsidies, It refuses to add a wrong subsidy', function () {
+    return agent.post('/subsidies')
+      .send(wrongSubsidy)
+      .then(function (res) {
+        expect(res).to.have.status(400);
+        expect(res).to.be.json;
+        expect(res.body).to.not.be.empty;
+      })
+      .catch(helpers.handler);
+  });  
+    
+  it('METHOD : GET, PATH : /subsidies, It returns a list of subsidies', function () {
+      return agent.get('/subsidies')
+        .then(function (res) {
+          expect(res).to.have.status(200);
+          expect(res).to.be.json;
+          expect(res.body).to.not.be.empty;
+          expect(res.body).to.have.length(1);
+        })
+        .catch(helpers.handler);
+    });
+  
+  it('METHOD : GET, PATH : /subsidies/:id, It returns one subsidy', function () {
```

Comment:
Duplicate tests aren't a terrible problem.  One caution though: if the POST method breaks, you may spend lots of time trying to debug the GET request rather than the POST request, since it will look like the GET request is failing as well.